src/i965_drv_video.c

```diff
@@ -6720,6 +6720,7 @@ i965_Terminate(VADriverContextP ctx)
     int i;
 
     if (i965) {
+#if HAVE_HYBRID_CODEC
```

Comment:
No, it is not strictly necessary. The reason I ran into this is because I'm compiling the intel driver as a static library and the invocation of `dlclose()` in this code segment was causing problems.

(I have a patched version of libva which allows it to be built and used statically with the intel driver embedded.)